### PR TITLE
Theoretical biology prep

### PR DESCRIPTION
**Pull request by drvinceknight**

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -38,12 +38,29 @@
...
-the definitions of extortionate strategies are algebraically rigid, apply only
-to memory-one strategies, and require complete knowledge of a strategy (memory-one
-cooperation probabilities). The contribution of this work  is a method to detect extortionate
-behaviour from the history of play of a strategy. When applied to a corpus of
+Establishing and maintaining mutual cooperation in agent to agent interactions
+can be considered as a question of direct reciprocity. This mechanism can be
+readily applied to the Iterated Prisoner's Dilemma. Agents cooperate, at a
+small cost to themselves, in the hope of obtaining a future benefit. In 2012
+a new type of strategy was introduced: zero-determinant strategies which
+made way for strategies that were provable extortionate.
```
Comment (marcharper) [inline main.tex]: provably

File: main.tex [hunk 2]
```diff
@@ -60,11 +77,11 @@
...
-The Iterated Prisoner's Dilemma is a model for rational and evolutionary
-interactive behaviour, having applications in biology, the study of human
-social behaviour, and many other domains. A standard representation  of the game
-is given in equation~\ref{eqn:theipd}, where the constraints ensure a non cooperative
-equilibrium.
+The Iterated Prisoner's Dilemma (IPD) is a model for rational and evolutionary
+interactive behaviour, having applications in biology, the study of human social
+behaviour, and many other domains. A standard representation  of the game is
+given in equation~\ref{eqn:theipd}, where the constraints ensure a non
+cooperative equilibrium.
```
Comment (marcharper) [inline main.tex]: non-cooperative

File: main.tex [hunk 3]
```diff
@@ -76,15 +93,22 @@
...
-
-Since the introduction of
-zero-determinant (ZD) strategies in \cite{Press2012}, extortionate strategies have
-received considerable interest in the literature \cite{hilbe2015partners}.
-These strategies ``enforce'' a difference in stationary
-payouts between themselves and their opponents. The definition requires a
-precise algebraic relationship between the probabilities of cooperation given
-the outcome of the previous round of play and slight alterations to these
-probabilities can cause a strategy to no longer satisfy the necessary equations.
+The non cooperative behaviour is no longer necessary when the game is repeated.
```
Comment (marcharper) [inline main.tex]: non-cooperative

File: main.tex [hunk 4]
```diff
@@ -76,15 +93,22 @@
...
+An initial major piece of research on this topic was the work
```
Comment (marcharper) [inline main.tex]: Maybe just say "Early work in the field showed that..."

File: main.tex [hunk 5]
```diff
@@ -76,15 +93,22 @@
...
+of~\cite{Axelrod1980, Axelrod1980a} which show that a cooperative behaviour
+which was able to respond to uncooperative actions: Tit For Tat would perform
```
Comment (marcharper) [inline main.tex:98]: "Tit-For-Tat" perhaps, let's just be sure to be consistent throughout

File: main.tex [hunk 6]
```diff
@@ -76,15 +93,22 @@
...
+strongly. The simplicity of Tit For Tat, which only requires knowledge of the
+opponents previous play lead to most research concentrating on these so called
+Memory 1 strategies.
```
Comment (marcharper) [inline main.tex]: memory-one

File: main.tex [hunk 7]
```diff
@@ -76,15 +93,22 @@
...
+
+One specific type of these strategies are zero-determinant (ZD) strategies which
+were introduced in \cite{Press2012}, extortionate strategies have received
```
Comment (marcharper) [inline main.tex]: Split on the comma. "Of these, extortionate ..."

File: main.tex [hunk 8]
```diff
@@ -76,15 +93,22 @@
...
-probabilities can cause a strategy to no longer satisfy the necessary equations.
+The non cooperative behaviour is no longer necessary when the game is repeated.
+An initial major piece of research on this topic was the work
+of~\cite{Axelrod1980, Axelrod1980a} which show that a cooperative behaviour
+which was able to respond to uncooperative actions: Tit For Tat would perform
+strongly. The simplicity of Tit For Tat, which only requires knowledge of the
+opponents previous play lead to most research concentrating on these so called
+Memory 1 strategies.
+
+One specific type of these strategies are zero-determinant (ZD) strategies which
```
Comment (marcharper) [inline main.tex]: A subclass of memory-one strategies known as zero-determinant ...

**Review by marcharper — APPROVED** [review]
Just a few minor comments. Thanks for updating and resubmitting the paper!

File: main.tex (hunk 4 above)
Comment (drvinceknight) [reply main.tex]: Yeah this is better.

**Comment by drvinceknight** [comment]
Thanks @marcharper, I'll make those changes and submit early next week :+1:

**Comment by Nikoleta-v3** [comment]
Thank you for your work @drvinceknight 🙏🏻

**Pull request merged**